mitosheet/src/mito/utils/keyboardShortcuts.tsx

```diff
@@ -126,6 +126,100 @@ export const keyboardShortcuts: KeyboardShorcut[] = [
             keys: ['h']
         },
         action: ActionEnum.OpenFindAndReplace
+    },
+    {
+        macKeyCombo: {
+            ctrlKey: true,
+            shiftKey: true,
+            keys: ['`', '~']
```

Comment:
Hmm. So this triggers if any key is in the list, then? That makes sense, but it feels like a little weird -- e.g. what happens if it's a keyboard shortcut that requires two keys?

Reply:
Yeah hm. Maybe I should change the name of this field. or maybe I should figure out a way to convert these symbols to "lowercase"